app/views/groups/assign_scans.html.erb

```diff
@@ -30,13 +30,20 @@
     };
     $("#assign_student").submit(function (e) {
       e.preventDefault();
-      $.ajax({
-        method: "GET",
-        url: "assign_student_and_next",
-        dataType: "json",
-        data: $("#assign_student").serialize(),
-        success: parseNextScan
-      });
+      const queryString = $("#assign_student").serialize();
+      const url = "assign_student_and_next";
+      const requestUrl = `${url}?${queryString}`;
+      fetch(requestUrl, {
+        headers: {
+          Accept: "application/json",
+        }
+      })
+      .then((response) =>{
```

Comment:
There's some whitespace formatting issues in this code block (note that prettier doesn't check `.erb` files, so you need to be careful with style yourself.